mitodl/micromasters
# Rename UWSGI_THREAD_COUNT and UWSGI_PROCESS_COUNT, and remove redundant if-not-env blocks

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #4597 

#### What's this PR do?
- Renames `UWSGI_THREAD_COUNT` to `UWSGI_THREADS` and `UWSGI_PROCESS_COUNT` to `UWSGI_PROCESSES`
- Removes `if-env` blocks because uWSGI will populate those variables automatically

#### How should this be manually tested?
First, rename `UWSGI_THREAD_COUNT` and `UWSGI_PROCESS_COUNT` if you have them defined in your `.env` file. Then go to the homepage and verify that nothing has broken.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
